lib/config.go

```diff
@@ -55,37 +57,40 @@ func (m *ConfigMethods) GetConfig(p *GetConfigParams, res *[]byte) (err error) {
 	if p.Field != "" {
 		encode, err = cfg.Get(p.Field)
 		if err != nil {
-			return fmt.Errorf("error getting %s from config: %s", p.Field, err)
+			return nil, fmt.Errorf("error getting %s from config: %s", p.Field, err)
```

Comment:
gotta wrap them errors:

```suggestion
			return nil, fmt.Errorf("error getting %s from config: %w", p.Field, err)
```